src/Tribe/Main.php

```diff
@@ -3826,76 +2990,6 @@ public function verify_unique_name( $name, $post_type ) {
 			return ( $results ) ? 0 : 1;
 		}
 
-		/**
-		 * Given a date (YYYY-MM-DD), returns the first of the next month
-		 * hat tip to Dan Bernadict for method cleanup
-		 *
-		 * @param string $date
-		 *
-		 * @return string Next month's date
-		 * @throws OverflowException
-		 */
-		public function nextMonth( $date ) {
```

Comment:
Brought back and deprecated. ([see here](https://github.com/the-events-calendar/the-events-calendar/pull/3788/files#diff-d954d62533e45649ed5be315a5d94782039510bc401eae4567ae2ee03aaaacc0R3139))